D_controllers.md

```diff
@@ -1,6 +1,7 @@
 Phoenix controllers act as intermediary modules. Their functions - called actions - are invoked from the router in response to HTTP requests. The actions, in turn, gather all the necessary data and perform all the necessary steps before - in a typical case - invoking the view layer to render a template.
 
-A newly generated Phoenix app will have a single controller, the `PageController`, which looks like this.
+A newly generated Phoenix app will have a single controller, the `PageController`, which can be found in
```

Comment:
I think we've been using "at" instead of "in" for, ". . . which can be found at . . . ".